src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  font-family: Arial;
```

Comment:
indicate subfont if Arial won't be loaded

Reply:
fixed